sqlglot/dialects/dialect.py

```diff
@@ -53,12 +53,20 @@ def __new__(cls, clsname, bases, attrs):
         klass.generator_class = getattr(klass, "Generator", Generator)
 
         klass.tokenizer = klass.tokenizer_class()
-        klass.quote_start, klass.quote_end = list(klass.tokenizer_class.QUOTES.items())[
-            0
-        ]
+        klass.quote_start, klass.quote_end = list(
+            klass.tokenizer_class._QUOTES.items()
+        )[0]
         klass.identifier_start, klass.identifier_end = list(
-            klass.tokenizer_class.IDENTIFIERS.items()
+            klass.tokenizer_class._IDENTIFIERS.items()
         )[0]
+        if len(klass.tokenizer_class._BIT_STRINGS) > 0:
```

Comment:
you can just do if bit_strings nince len(x) > 0 is truthy

Reply:
Fixed